src/containers/my-cooperations/cooperation-details/CooperationDetails.styles.ts

```diff
@@ -1,20 +1,40 @@
 import palette from '~/styles/app-theme/app.pallete'
+import { breakpointsTheme } from '~/styles/app-theme/custom-mui.styles'
 
 export const styles = {
   tabsWrapper: {
     display: 'flex',
     justifyContent: 'space-between'
   },
-  banner: {
+  pageContent: {
+    width: { md: '1340px', sm: '980px' }
+  },
+  banner: (isNotesOpen: boolean) => ({
     display: 'flex',
     alignItems: 'center',
     cursor: 'pointer',
-    color: palette.basic.blueGray,
-    mb: '20px'
-  },
-  notes: {
-    py: '10px',
-    color: 'inherit'
+    mb: '10px',
+    color: isNotesOpen ? palette.basic.lightBlue : palette.basic.blueGray,
+    transform: isNotesOpen ? 'translateX(-200px)' : 'translateX(0)',
+    transition: 'transform 0.3s ease-in-out, color 0.3s ease-in-out',
```

Comment:
Try to use out <AppDrawer /> component for notes container, and you will avoid transition and some other styles

Reply:
> Try to use our 'AppDrawer' component for notes container, and you will avoid transition and some other styles

used AppDrawer